templates/components/search/displaypreference_list.html.twig

```diff
@@ -0,0 +1,108 @@
+{#
+ # ---------------------------------------------------------------------
+ #
+ # GLPI - Gestionnaire Libre de Parc Informatique
+ #
+ # http://glpi-project.org
+ #
+ # @copyright 2015-2023 Teclib' and contributors.
+ # @copyright 2003-2014 by the INDEPNET Development Team.
+ # @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ #
+ # ---------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of GLPI.
+ #
+ # This program is free software: you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation, either version 3 of the License, or
+ # (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+
+{% import 'components/form/basic_inputs_macros.html.twig' as inputs %}
+
+{% if preferences|length > 0 %}
+   <div class="mb-3">
+      {% if massiveactionparams['specific_actions']|length > 0 %}
+         <form id="massDisplayPreference{{ rand }}" method="get" action="{{ path('front/massiveaction.php') }}"
+            data-search-itemtype="DisplayPreference" data-submit-once>
+         {% do call('Html::showMassiveActions', [massiveactionparams]) %}
+      {% endif %}
+      {{ inputs.hidden('users_id', users_id, {
+         'data-glpicore-ma-tags': 'common'
+      }) }}
+      <table class="table table-striped display-prefs-list">
+         <thead>
+            <tr>
+               <th style="width: 10px">
+                  {% if massiveactionparams['specific_actions']|length > 0 %}
+                     {{ call('Html::getCheckAllAsCheckbox', ['massDisplayPreference' ~ rand])|raw }}
+                  {% endif %}
+               </th>
+               <th>{{ _n('Type', 'Types', 1) }}</th>
+               <th class="text-end">{{ __('Number of modifiable columns') }}</th>
```

Comment:
Not caused by this pr, I'm not sure about the purpose of this column. Seems to be the number of columns set by the user for an itemtype.
So the label seems wrong but whatever, I think we should drop completly this th, the feature is useless. 

If we do this (the removal), move to ul>li design, as having a table for one columns is useless

Reply:
I don't know. The column always existed, but without a label. It currently shows the number of columns in the saved preferences. That would include any added by default or by a user, but not fixed columns which are always shown.

I don't know if the original purpose was just to show how "heavy" each list was or some other reason.